uint32 y_size
float64[] y

uint32 C_size
float64[] C

uint32 R_size
float64[] R

uint32 x_bar_size
float64[] x_bar

uint32 P_bar_size
float64[] P_bar

geometry_msgs/Pose allocation
